Doc/c-api/type.rst

```diff
@@ -557,20 +557,20 @@ The following functions and structs are used to create
       To avoid issues, use the *bases* argument of
       :c:func:`PyType_FromSpecWithBases` instead.
 
-     .. versionchanged:: 3.9
+      .. versionchanged:: 3.9
 
-        Slots in :c:type:`PyBufferProcs` may be set in the unlimited API.
+         Slots in :c:type:`PyBufferProcs` may be set in the unlimited API.
```

Comment:
There shouldn't be a newline between `.. versionchanged` and what has changed (the previous text was already wrong). See https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded.